### PR TITLE
Setup HelloWord

### PR DESCRIPTION
**Pull request by NeckerFree**
Request for Setup Hello World

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Ignore MD test file
+test.MD
```
Comment (keroloussamy) [inline .gitignore]: - [x] Kindly add `node_modules/` to .gitignore file as you did with test.md file, to not upload all the packages to your repo.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @NeckerFree, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Implement all the requirements.  ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (NeckerFree) [reply .gitignore]: Ok done, thanks Kerolous.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+# Ignore node_modules folder
+node_modules
```
Comment (BenMukebo) [inline .gitignore]: - [ ] Great job on adding the node_modules file to the .gitignore file. however, the file is still visible in the repository, 
Screen Shot 2022-02-09 at 17 32 02

the reason for this is that you created the file and committed it before adding it to the .gitigone file. to undo this you can run this command `git rm --cached node_modules`. then you can add and commit. with that, the file will not be available in the repository. After this, ou are good to go 😄

**Review by BenMukebo — CHANGES_REQUESTED** [review]
Hi  @Elio Cortés Martínez, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻  


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 2 above)
Comment (NeckerFree) [reply .gitignore]: Hi @BenMukebo, I have done the change you suggested, thanks

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @NeckerFree ,  

Your project is complete! Congratulations!  🎉💯🌟 


Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**